### PR TITLE
disabling make built-in rules to speed up execution

### PR DESCRIPTION
**Pull request by ochriste**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -294,6 +294,162 @@ ENVIRONMENT_VARS += INSTANCE_ID=${INSTANCE_ID} \
 C2C_TEMPLATE_CMD = $(ENVIRONMENT_VARS) $(VENV_BIN)/c2c-template --vars $(VARS_FILE)
 MAKE_FILES = $(shell ls -1 *.mk) CONST_Makefile
 
+# Disabling Make Implicit Rules to speed up execution time
+.SUFFIXES:
+
+# Disabling Make Implicit Rules to speed up execution time
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Implicit => Built-in

**Comment by sbrunner** [comment]
Just a small comment, otherwise that looks good :-)

**Comment by fredj** [comment]
According to the documentation `.SUFFIXES:` deletes the default suffixes but why `SUFFIXES :=` is sets just after that ?

**Comment by ochriste** [comment]
we made some tests with sbrunner and the execution time was even shorter when overriding the list.

but it seems this has broken the variable replacement system
we need to make some more tests

**Comment by ochriste** [comment]
removed SUFFIXES := as it was causing issue with some tasks execution

**Comment by sbrunner** [comment]
But is steal not `Implicit Rules` but `built-in rules`

**Comment by sbrunner** [comment]
Excellent, merge :-)

**Pull request merged**